packages/server/src/hono/routes/memory.ts

```diff
@@ -86,7 +86,33 @@ export function createMemoryRouter(getAgent: () => DextoAgent) {
         },
     });
     app.openapi(createMemoryRoute, async (ctx) => {
-        const input = ctx.req.valid('json');
+        console.log('[MEMORY DEBUG] Route handler called');
+        console.log('[MEMORY DEBUG] Request headers:', ctx.req.header());
+        console.log('[MEMORY DEBUG] Request method:', ctx.req.method);
+        console.log('[MEMORY DEBUG] Request bodyUsed:', (ctx.req.raw as any).bodyUsed);
+        console.log('[MEMORY DEBUG] Request body:', (ctx.req.raw as any).body);
```

Comment:
Remove all of these debug logs we used in this function. This was just for testing and fixing a bug.